### PR TITLE
Feat/develop/wire fixes

### PR DESCRIPTION
**Pull request by ShowriSrinivas**
# Changes

1. Wire transfer international fixes
2. Implemented security options
3. Added Payment details

# Screenshots

Screenshot from 2023-02-15 13-03-40

### CODE REVIEW

File: app/src/lib/hooks/useCreatePayment.ts [hunk 1]
```diff
@@ -201,18 +201,22 @@ export const useCreatePayment = (paymentInfo: PaymentData | undefined, orgId: st
...
+      wireData.routingNumber = paymentInfo?.wireData?.routingNumber;
       inputData.wireData = {
         ...wireData,
         billingDetails: copiedBillingDetails,
       };
     } else {
-      delete wireData.accountNumber;
-      delete wireData.routingNumber;
-      delete wireData.country;
+      wireData.iban = (paymentInfo?.wireData?.bankAddress?.country ?? '') + (paymentInfo?.wireData?.iban ?? '');
```
Comment (alexinx) [inline app/src/lib/hooks/useCreatePayment.ts:219]: we should pass country code, not country,
is it tested?

File: app/src/lib/hooks/useCreatePayment.ts [hunk 2]
```diff
@@ -201,18 +201,22 @@ export const useCreatePayment = (paymentInfo: PaymentData | undefined, orgId: st
     delete copiedBillingDetails.firstName;
     delete copiedBillingDetails.lastName;
     inputData.paymentType = 'Wire';
-    const wireData = { ...paymentInfo?.wireData };
+    const wireData:any = {
+      bankAddress: {
+        bankName: paymentInfo?.wireData?.bankAddress?.bankName,
+        country: paymentInfo?.wireData?.bankAddress?.country,
+        city: paymentInfo?.wireData?.bankAddress?.bankName,
```
Comment (alexinx) [inline app/src/lib/hooks/useCreatePayment.ts:208]:
```suggestion
        city: paymentInfo?.wireData?.bankAddress?.city,
```

File: app/src/lib/hooks/useCreatePayment.ts (hunk 1 above)
Comment (ShowriSrinivas) [reply app/src/lib/hooks/useCreatePayment.ts:219]: We store code only

**Comment by alexinx** [comment]
can you plz attach payment confirmation screen for wire

File: app/src/lib/hooks/useCreatePayment.ts (hunk 2 above)
Comment (ShowriSrinivas) [reply app/src/lib/hooks/useCreatePayment.ts]: Done

File: app/src/lib/hooks/useCreatePayment.ts [hunk 3]
```diff
@@ -150,7 +151,7 @@ export const useCreatePayment = (paymentInfo: PaymentData | undefined, orgId: st
           },
         },
       });
-      await createPayment({
+      const result1 = await createPayment({
```
Comment (alexinx) [inline app/src/lib/hooks/useCreatePayment.ts:154]:
```suggestion
      const paymentResponse = await createPayment({
```

File: app/src/lib/views/PaymentConfirmation/PaymentDetailsView.tsx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+import React, { useMemo } from 'react';
+import { usePaymentInfo } from '../../hooks';
+import { MixTheme } from '../../theme';
+import RowItem from './RowItem';
+
+const PaymentDetailsView = () => {
+  const theme = useTheme<MixTheme>();
+  const { paymentResult } = usePaymentInfo();
+
+  const style = useMemo(() => {
```
Comment (alexinx) [inline app/src/lib/views/PaymentConfirmation/PaymentDetailsView.tsx:11]: good

File: app/src/lib/service/CookieService.ts [hunk 5]
```diff
@@ -34,4 +34,5 @@ export const CookieService = {
   taxes: new CookieStorage('taxes'),
   reserveLotData: new CookieStorage('reserveLotData'),
   collectionData: new CookieStorage('collectionData'),
+  paymentResult: new CookieStorage('paymentResult'),
```
Comment (alexinx) [inline app/src/lib/service/CookieService.ts:37]: do we need this data?

Comment (ShowriSrinivas) [reply app/src/lib/service/CookieService.ts:37]: Yes, as explained to show wire data details

File: app/src/lib/hooks/useCreatePayment.ts (hunk 3 above)
Comment (ShowriSrinivas) [reply app/src/lib/hooks/useCreatePayment.ts:154]: Done

**Review by alexinx — APPROVED** [review]

**Pull request merged**